### ISSUE TITLE
jsx-handler-names false positive on A tag

### ISSUE THREAD

**Issue by RusAlex** [yannickcr/eslint-plugin-react#919]
It must not require to rename handler in next snippet

``` js
<a href="#" onClick={clickHandler} >Link</a>
```

**Comment by ljharb** [comment]
The point of the rule is so you name your identifier something like `handleClick` - why would this be different here?

**Comment by RusAlex** [comment]
Because its native event for DOM element?

I just want to know how to sort this correctly.

**Comment by ljharb** [comment]
If you rename your callback identifier from `clickHandler` to `handleClick`, i believe it will pass.
